core/Enclave/enclave_hsm.cpp

```diff
@@ -458,27 +496,32 @@ sgx_status_t enclave_rsa_encrypt(const uint8_t *cmk_blob,
                                  uint8_t keyspec,
```

Comment:
ciphertext_len, keyspec, cmk_blob_size not used?

Reply:
you need to check the ciphertext_len by the keyspec.

Reply:
ciphertext_len, keyspec and  cmk_blob_size have been used and ciphertext_len is going to be checked by keyspec